src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
Alright they use it to prevent unused parameter AND unused local variable warnings, but I doubt that they use it for return values.